samples/test/create-custom-job.test.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {after, describe, it} = require('mocha');
+const uuid = require('uuid').v4;
+const cp = require('child_process');
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+const customJobDisplayName = `temp_create_custom_job_test${uuid()}`;
+const containerImageUri = 'gcr.io/ucaip-test/ucaip-training-test:latest';
+const project = process.env.CAIP_PROJECT_ID;
```

Comment:
The fact that we're referencing `CAIP_PROJECT_ID` here, makes me think that we don't actually need to change the value of `GCLOUD_PROJECT` the following:

https://github.com/googleapis/nodejs-ai-platform/blob/master/.kokoro/pre-samples-test.sh

Is populating the `CAIP_PROJECT_ID` variable with an appropriate value -- if not we should ensure that it is.

In Q1 I'm going to make an effort to make dropping in additional environment variables like this easier.

Reply:
Maybe I'm wrong about this, but I don't think it is. Before changing the GCLOUD_PROJECT the tests would all fail because they were being run against the wrong project.

Reply:
@lucaswadedavis the tests are passing the value of `CAIP_PROJECT_ID` as the project identifier to the `sample`, so I believe the only reason they'd be failing would be if that `project` value is being disregarded in the sample.